kanidmd/lib/src/idm/account.rs

```diff
@@ -156,7 +156,7 @@ impl Account {
     #[instrument(level = "trace", skip_all)]
     pub(crate) fn try_from_entry_ro(
         value: &Entry<EntrySealed, EntryCommitted>,
-        qs: &mut QueryServerReadTransaction,
+        qs: &QueryServerReadTransaction,
```

Comment:
please leave the &mut here for now, as I'm working on some cleanups that will require it to be present (unless it generates a warning, then leave it removed )

Reply:
Hm, if I keep the reference mutable, it requires a pretty large refactoring (adding a lot of `mut`s all the way to the top) and then causes some borrowing issues. I'm sure there's a better way but I don't know rust well enough to come up with a good solution

Reply:
All good, leave it as is for now then :)